cmd/apprepository-controller/pkg/apis/apprepository/v1alpha1/types.go

```diff
@@ -36,11 +36,15 @@ type AppRepository struct {
 
 // AppRepositorySpec is the spec for an AppRepository resource
 type AppRepositorySpec struct {
-	Type               string                 `json:"type"`
-	URL                string                 `json:"url"`
-	Auth               AppRepositoryAuth      `json:"auth,omitempty"`
-	ResyncRequests     uint                   `json:"resyncRequests"`
-	SyncJobPodTemplate corev1.PodTemplateSpec `json:"syncJobPodTemplate"`
+	Type string            `json:"type"`
+	URL  string            `json:"url"`
+	Auth AppRepositoryAuth `json:"auth,omitempty"`
+	// DockerRegistrySecrets is a list of dockerconfigjson secrets which exist
+	// in the same namespace as the AppRepository and should be included
+	// automatically for matching images.
+	DockerRegistrySecrets []string               `json:"dockerRegistrySecrets,omitempty"`
```

Comment:
nit: I would move this param to the last (at first sight I thought you were adding 3 new params to the spec)

Reply:
Done.